src/pages/RFOX/hooks/useStakingInfoQuery.ts

```diff
@@ -63,5 +63,5 @@ export const useStakingInfoQuery = <SelectData = StakingInfo>({
     select,
   })
 
-  return stakingInfoQuery
+  return { ...stakingInfoQuery, queryKey }
```

Comment:
Not using `wagmi/query`'s `useQuery` variant anymore, so we expose queryKey the same like `wagmi/query` does